serverless.yml

```diff
@@ -5,9 +5,9 @@ plugins:
 
 provider:
   name: aws
-  runtime: ruby2.5
+  runtime: ruby2.7
   stage: ${opt:stage, "development"}
-  profile: se-dev
+  profile: ${env:AWS_PROFILE}
 
```

Comment:
Can you test to see if this even needed in the yaml file? Can you just set AWS_PROFILE environment variable and remove this part?